api/responses/get-recommendation.json

```diff
@@ -85,5 +85,44 @@
     "name": "Paula Smith",
     "firstName": "Paula",
     "surname": "Smith"
+  },
+  "hasVictimsInContactScheme": {
+    "selected": "YES",
+    "allOptions": [
+      { "value": "YES", "text": "Yes" },
+      { "value": "NO", "text": "No" },
+      { "value": "NOT_APPLICABLE", "text": "Not applicable" }
```

Comment:
Hi Jon, 
Is this better as N/A to be consistent with the offender management object?

Reply:
We already have 'Not applicable' for Victim contact scheme. Let me check with Toria

Reply:
done